File: 240116/10개 입력받기/receive-10-inputs.c
#include <stdio.h>

int main() {
    // 여기에 코드를 작성해주세요.
    int sum=0;
    int cnt=10;
    int arr[10];
    for (int i=0;i<10;i++){
        scanf("%d",&arr[i]);
        if (arr[i]==0){
            cnt=i;
            break;
        }
        sum+=arr[i];
        
    }
    printf("%d %.1lf",sum,(double)sum/cnt);
    return 0;
}

File: 240312/10개 입력받기/receive-10-inputs.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc=new Scanner(System.in);
        int sum=0;
        double n=0;
        int[] arr=new int[10];
        for (int i=0;i<10;i++){
            arr[i]=sc.nextInt();
            if (arr[i]==0){
                break;
            }
            sum+=arr[i];
            n+=1;
        }
        System.out.printf("%d %.1f",sum,sum/n);
    }
}